api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/domain/CreateAccountRequest.java

```diff
@@ -29,5 +29,9 @@ public class CreateAccountRequest {
     private String tenableAccessKey;
     private String tenableSecretKey;
 
+    private String redhatAccountId;
```

Comment:
Why do we have a single CreateAccountRequest for all available plugins. Why can't we have a separate class for every plugin; e.g., 
RedhatCreateAccountRequest {
private String redhatAccountId;
private String redhatAccountName;
private String redhatToken;
private String owner;
}

Reply:
yes but I had to use the interface **AccountsService** to follow the common format